WordPress/src/test/java/org/wordpress/android/ui/jetpack/scan/ScanViewModelTest.kt

```diff
@@ -517,22 +518,23 @@ class ScanViewModelTest : BaseUnitTest() {
             verify(fetchScanStateUseCase, times(2)).fetchScanState(site = site)
         }
 
-    @Test
-    fun `given threats are fixing, when threats fix status is checked, then an indeterminate progress bar is shown`() =
-        test {
-            whenever(fetchFixThreatsStatusUseCase.fetchFixThreatsStatus(any(), any())).thenReturn(
-                flowOf(FetchFixThreatsState.InProgress(mock()))
-            )
-            val observers = init()
-
-            fetchFixThreatsStatus(observers)
-
-            val indeterminateProgressBars = (observers.uiStates.last() as ContentUiState).items
-                .filterIsInstance<ProgressState>()
-                .filter { it.isIndeterminate && it.isVisible }
-
-            assertThat(indeterminateProgressBars.isNotEmpty()).isTrue
-        }
+    // TODO ashiagr fix test
```

Comment:
Is this intentionally commented out or were you planning on getting back to it in this PR?

Reply:
Oh yes, I wanted to fix it here. Thanks for pointing it.

I've removed test cases related to updating button and progress bar on threats fixing in 7e1742e as threats fixing content building is now done in the builder class.